client/component/SelectionConfig.tsx

```diff
@@ -127,6 +128,25 @@ export function SelectionConfig(props: SelectionConfigProps) {
     }
   }
 
+  function handleDeleteSelection() {
+    if (props.currentSelectionID) {
+      const selection = props.selections.find(
+        (s) => s.id === props.currentSelectionID
+      );
+      const filteredSelections = props.selections.filter(
+        (s) => s.id !== props.currentSelectionID
+      );
+      if (selection) {
+        props.updateSelections(selection, true, true);
+        if (filteredSelections.length > 0) {
```

Comment:
Use findLast instead

Reply:
I can't get `findLast` working. Related to this: https://github.com/microsoft/TypeScript/issues/48829.

Reply:
It's a method on any array: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findLast

Reply:
Yes, but trying to use it brings up an error: `Property 'findLast' does not exist`.

Reply:
I tried updating the target but not got it working yet.

Reply:
Have you reinstalled your node modules recently? Maybe `npm i -f` and restart code.

Reply:
I can get it to choose the last selection correctly using `findLast` now but eslint does bring up compiler issues.